src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherCommand.cs

```diff
@@ -45,11 +46,31 @@ public class GetAzureNetworkWatcherCommand : NetworkWatcherBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string ResourceGroupName { get; set; }
 
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "Location of the network watcher.",
+            ParameterSetName = "SetByLocation")]
```

Comment:
@junyezhu nit: should this be "GetByLocation" rather than "SetByLocation"?

This comment applies to all instances of this being added

Reply:
I think Set makes sense, since we need to first set the resource, then we can add, get or remove.

Reply:
@junyezhu I'm not sure I understand what you mean by "first set the resource" -- the parameter set name is what the users see in the help content for the cmdlet, and if they see that providing a `Location` will "set" instead of "get", which the name of this cmdlet implies it does, then that might be a confusing experience.

Reply:
I don't think it will be confusing, one may imply this as set the command request by Location.  More importantly, every similar cases in network watcher commands are using setBy. There will be a lot of places need to be updated, and I don't think this is an error, more like an enhancement. I prefer not to do that in this pr.

Reply:
@cormacpayne  Could you respond this issue? Thanks